pom.xml

```diff
@@ -33,35 +33,240 @@
+        <maven-install-plugin.version>2.5.2</maven-install-plugin.version>
+        <maven-jacoco-plugin.version>0.8.1</maven-jacoco-plugin.version>
+        <maven-jaxb2-plugin.version>0.8.2</maven-jaxb2-plugin.version>
+        <maven-source-plugin.version>3.0.1</maven-source-plugin.version>
+        <maven-war-plugin.version>2.2</maven-war-plugin.version>
+        <!-- Mina version is not the latest version due to Apache FTPServer 1.0.6 not supporting newer releases properly -->
+        <mina.version>2.0.6</mina.version>
+        <mockito-core.version>2.8.47</mockito-core.version>
+        <mockito-all.version>1.10.19</mockito-all.version>
+        <moment.version>2.20.1</moment.version>
+        <mpegts-streamer.version>0.1.0_2</mpegts-streamer.version>
+        <net.minidev.asm.version>1.0.2</net.minidev.asm.version>
+        <netty.version>4.1.16.Final</netty.version>
+        <nitf-imaging.version>0.8.1</nitf-imaging.version>
+        <node.version>v8.9.4</node.version>
+        <objenesis.version>2.5.1</objenesis.version>
+        <ogc-filter-v_1_1_0-schema.version>1.1.0_5</ogc-filter-v_1_1_0-schema.version>
+        <ogc-tools-gml-jts.version>1.0.3</ogc-tools-gml-jts.version>
+        <ognl.version>2.6.9</ognl.version>
+        <opendj-embedded.app.version>1.3.3</opendj-embedded.app.version>
+        <opengis.bundle.version>${opengis.version}_1</opengis.bundle.version>
+        <opengis.version>19.1</opengis.version>
+        <openlayers.version>4.6.5</openlayers.version>
+        <opensaml.osgi.version>3.3.0_2</opensaml.osgi.version>
+        <oracle.jdbc.version>11.2.0.4.0</oracle.jdbc.version>
+        <oracle.version>11.2.0</oracle.version>
+        <org.geotools.bundle.version>${org.geotools.version}_2</org.geotools.bundle.version>
+        <org.geotools.version>19.1</org.geotools.version>
+        <org.ops4j.pax.swissbox.version>1.8.0</org.ops4j.pax.swissbox.version>
+        <org.ow2.asm.version>5.0.2</org.ow2.asm.version>
+        <org.rrd4j.version>2.2</org.rrd4j.version>
+        <org.slf4j.version>1.7.1</org.slf4j.version>
+        <osgi.version>5.0.0</osgi.version>
+        <owasp-html-sanitizer.version>20171016.1</owasp-html-sanitizer.version>
+        <ows-v_1_0_0-schema.version>1.1.0</ows-v_1_0_0-schema.version>
+        <pax.cdi.version>0.12.0</pax.cdi.version>
+        <pax.exam.version>4.11.0</pax.exam.version>
+        <pax.url.version>2.5.2</pax.url.version>
+        <pax.web.version>6.0.11</pax.web.version>
+        <pdfbox.version>2.0.11</pdfbox.version>
+        <picocontainer.version>1.3_1</picocontainer.version>
+        <plexus-compiler-javac-errorprone.version>2.8.2</plexus-compiler-javac-errorprone.version>
+        <poi.contrib.version>3.7-beta3</poi.contrib.version>
+        <poi.version>3.17</poi.version>
+        <powermock.version>1.6.4</powermock.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
❓Should we keep the `project.` properties separate since they are simple string replacements and not versions?

Reply:
Separate in this pom or separate as in the downstream projects?